website/docs/components/copy/button/partials/specifications/spacing.md

```diff
@@ -0,0 +1,5 @@
+## Spacing
+
+When wrapping an input control and CopyButton in a container, set the gap between each element to 8px.
```

Comment:
I'd argue that this is different from the other specifications and that this not purely informational because it's the responsibility of the consumer to do this.